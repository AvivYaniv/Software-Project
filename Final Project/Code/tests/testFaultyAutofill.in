solve tests/b3_3_faltyAutofill.txt
print_board
num_solutions
validate
autofill
validate