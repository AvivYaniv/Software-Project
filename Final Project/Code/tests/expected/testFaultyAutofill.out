Sudoku
------
Enter your command:
----------------------------------------
|      5.    |  3   6.    |  4.  1.  2.|
|      7.  1.|  9.  5     |          3.|
|  3.        |  8.        |  9.  7.  6.|
----------------------------------------
|          6.|  5.  9.    |  2.      8.|
|          4.|  2.  1     |  7.  6.  9.|
|      9.  3.|      8.    |          1.|
----------------------------------------
|  1.  2.  9.|  4.      6.|      8.  7.|
|  8.  3   5.|      7.  9.|  6.  2.  4.|
|      6.  7.|      2.  8.|  1.  9.  5.|
----------------------------------------
Enter your command:
----------------------------------------
|      5.    |  3   6.    |  4.  1.  2.|
|      7.  1.|  9.  5     |          3.|
|  3.        |  8.        |  9.  7.  6.|
----------------------------------------
|          6.|  5.  9.    |  2.      8.|
|          4.|  2.  1     |  7.  6.  9.|
|      9.  3.|      8.    |          1.|
----------------------------------------
|  1.  2.  9.|  4.      6.|      8.  7.|
|  8.  3   5.|      7.  9.|  6.  2.  4.|
|      6.  7.|      2.  8.|  1.  9.  5.|
----------------------------------------
Enter your command:
Number of solutions: 0
Enter your command:
Validation failed: board is unsolvable
Enter your command:
Cell <1,1> set to 9
Cell <3,1> set to 8
Cell <6,1> set to 7
Cell <7,2> set to 8
Cell <2,3> set to 4
Cell <3,3> set to 2
Cell <5,3> set to 4
Cell <1,4> set to 7
Cell <2,4> set to 1
Cell <1,5> set to 5
Cell <2,5> set to 8
Cell <6,5> set to 3
Cell <7,6> set to 5
Cell <5,7> set to 3
Cell <7,7> set to 3
Cell <4,8> set to 1
Cell <1,9> set to 4
----------------------------------------
|  9   5.  8 |  3   6.  7 |  4.  1.  2.|
|      7.  1.|  9.  5     |  8       3.|
|  3.  4*  2 |  8.  4*    |  9.  7.  6.|
----------------------------------------
|  7   1   6.|  5.  9.    |  2.      8.|
|  5   8   4.|  2.  1   3 |  7.  6.  9.|
|      9.  3.|      8.    |  5       1.|
----------------------------------------
|  1.  2.  9.|  4.  3*  6.|  3*  8.  7.|
|  8.  3   5.|  1   7.  9.|  6.  2.  4.|
|  4   6.  7.|      2.  8.|  1.  9.  5.|
----------------------------------------
Enter your command:
Error: board contains erroneous values
Enter your command:
Exiting...
